src/styles/blocks/body.scss

```diff
@@ -0,0 +1,56 @@
+@import "../utils/variebles";
+
+body {
+  display: flex;
+  height: $height-container;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: ($width-day * 7) + ($between-gap * 6);
+  margin: auto;
+
+  .day {
+    position: relative;
+    top: 0;
+    transition: top 0.5s ease;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $height-day;
+    width: $width-day;
+    border: 1px solid $color-black;
+    background-color: $color-grey;
+    cursor: pointer;
+
+    @each $index, $name in $weekday {
+      @if $index == 2 {
```

Comment:
do not use hardcoded in CSS value (`2`) - there should be modificator in `container` (should be named calendar):
![image](https://github.com/mate-academy/layout_calendar/assets/86923089/e1ee3c05-bd4c-4db4-b6af-a5bbaca99986)

Reply:
done